### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RuslanMelnykFE**
- [DEMO LINK](https://RuslanMelnykFE.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://RuslanMelnykFE.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Unfortunately, couldn't open the demo and test report link

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Need to fix it
image
image

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+.label--search {
+  position: relative;
+}
+
+.label--search::after {
+  content: "";
+  display: inline-block;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
```
Comment (anastasiiavorobiova) [inline src/style.css:68]:
```suggestion
  transform: translateY(-52%);
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  background: url("./images/Search.svg") no-repeat center center;
+  background-size: cover;
+}
+
+.form--big .label--search:after {
+  width: 19px;
+  height: 19px;
+  left: 24px;
```
Comment (anastasiiavorobiova) [inline src/style.css:76]:
```suggestion
  left: 26px;
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+}
+
+.form {
+  width: 100%;
+  margin-top: 20px;
+  padding: 0 10px;
```
Comment (anastasiiavorobiova) [inline src/style.css:21]:
```suggestion
  padding: 0 8px;
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done: consider continuing fixing tests for small input and styling focus styles accordingly to the Figma design: 
Screenshot 2023-08-13 at 16 59 01

**Review by volodymyr-soltys97 — APPROVED** [review]

**Pull request closed without merging**